tests/manual/table.js

```diff
@@ -9,10 +9,11 @@ import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor'
 import ArticlePluginSet from '@ckeditor/ckeditor5-core/tests/_utils/articlepluginset';
 import Table from '../../src/table';
 import TableToolbar from '../../src/tabletoolbar';
+import BlockQuote from '@ckeditor/ckeditor5-block-quote/src/blockquote';
 
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
-		plugins: [ ArticlePluginSet, Table, TableToolbar ],
+		plugins: [ ArticlePluginSet, Table, TableToolbar, BlockQuote ],
```

Comment:
Why was this change made? If you want to have a playground to test table with block quotes we'd also need this button in the toolbar.

Reply:
![image](https://user-images.githubusercontent.com/2270764/46272668-635f9a00-c552-11e8-8b26-8de6e4342845.png)

The button is already added. https://github.com/ckeditor/ckeditor5-table/pull/129/files#diff-4097acfafae638fcb639ff3a9d047f3cR18

Reply:
Yea, I was also a little surprised. I wanted to revert my invalid change than I saw the button :D